scripts/build/builders/android.py

```diff
@@ -520,6 +521,9 @@ def build_outputs(self):
                 "CHIPController.jar": os.path.join(
                     self.output_dir, "lib", "src/controller/java/CHIPController.jar"
                 ),
+                "libChipTlv.jar": os.path.join(
```

Comment:
Why the different spellings of "chip" here and in the path?

Reply:
Typo. since we use * it probably got lost ... fixing...